cspell.config.yaml

```diff
@@ -0,0 +1,27 @@
+---
+$schema: https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json
+version: '0.2'
+# Allows things like stringLength
+allowCompoundWords: true
```

Comment:
are both uses of allow compound words necessary?

Reply:
Fixed